### PR TITLE
3910 Standing Pretrial Notice PDF

### PR DESCRIPTION
**Pull request by wbyoko**
Screenshot 2020-01-28 09 50 56
Screenshot 2020-01-28 09 51 10

### CODE REVIEW

File: shared/src/business/useCaseHelper/standingPretrialNotice/generateStandingPretrialNoticeTemplate.js [hunk 1]
```diff
@@ -25,18 +25,28 @@ const generateStandingPretrialNoticeTemplate = async ({
...
   trialInfo.fullStartDate = formatDateString(
     trialInfo.startDate,
     'dddd, MMMM D, YYYY',
   );
   trialInfo.startDate = formatDateString(trialInfo.startDate, 'MMDDYYYY');
 
+  let respondentContactText = 'not available at this time';
+  if (trialInfo.respondents && trialInfo.respondents.length) {
+    const fristRespondent = trialInfo.respondents[0];
+    respondentContactText = `${fristRespondent.name} (${fristRespondent.contact.phone})`;
```
Comment (matthopson) [inline shared/src/business/useCaseHelper/standingPretrialNotice/generateStandingPretrialNoticeTemplate.js]: `frist` typo

File: shared/src/business/useCases/trialSessions/generateStandingPretrialNoticeInteractor.js [hunk 2]
```diff
@@ -42,8 +42,7 @@ exports.generateStandingPretrialNoticeInteractor = async ({
 
   // TODO - GET Respondent contact
```
Comment (rachaelparris) [inline shared/src/business/useCases/trialSessions/generateStandingPretrialNoticeInteractor.js]: Can this TODO be removed?

File: shared/src/business/useCaseHelper/standingPretrialNotice/generateStandingPretrialNoticeTemplate.js [hunk 3]
```diff
@@ -25,18 +25,28 @@ const generateStandingPretrialNoticeTemplate = async ({
...
+  trialInfo.startDay = formatDateString(trialInfo.startDate, 'dddd');
   trialInfo.fullStartDate = formatDateString(
     trialInfo.startDate,
     'dddd, MMMM D, YYYY',
   );
   trialInfo.startDate = formatDateString(trialInfo.startDate, 'MMDDYYYY');
 
+  let respondentContactText = 'not available at this time';
+  if (trialInfo.respondents && trialInfo.respondents.length) {
+    const fristRespondent = trialInfo.respondents[0];
```
Comment (rachaelparris) [inline shared/src/business/useCaseHelper/standingPretrialNotice/generateStandingPretrialNoticeTemplate.js]: `fristRespondent` -> `firstRespondent`

**Review by rachaelparris — APPROVED** [review]
A couple small things that I won't hold up the PR for. 😄

**Review by matthopson — APPROVED** [review]

**Pull request merged**